SPECIFICATION.md

```diff
@@ -34,7 +45,9 @@ TODO: Add JSON Schema reference here.
 
 The definitions object allows specification of reusable policies and calculations that can be referenced by events.
 
-**Support Policy Fields:**
+**Support**
+
+This is a list of support policies provided for a specific version or version range of a component. Note this SHOULD NOT include third party support policies or options.
```

Comment:
why should not, if the maintainers have a list of preferred third party options?

Reply:
```
The support object defines the support policies provided for a specific version or version range of a component. This may include first-party manufacturer support or third-party support options endorsed by the manufacturer.
```

how does it look?